src/test/regress/expected/multi_cluster_management.out

```diff
@@ -126,6 +165,21 @@ SELECT master_get_active_worker_nodes();
  (localhost,57637)
 (1 row)
 
+-- get get active nodes
+SELECT * from citus_nodes where active = 't';
+ nodename  | nodeport |    role     | active
+---------------------------------------------------------------------
+ localhost |    57636 | coordinator | t
+ localhost |    57637 | worker      | t
+(2 rows)
+
+-- get get inactive nodes
+SELECT * from citus_nodes where active = 'f';
+ nodename  | nodeport |  role  | active
+---------------------------------------------------------------------
+ localhost |    57638 | worker | f
+(1 row)
+
 -- add some shard placements to the cluster
 SET citus.shard_count TO 16;
```

Comment:
optionally, it can be nicer if we can add a test with a non-super user

```sql
CREATE ROLE alper;
SET ROLE alper;

-- make sure it works
SELECT * FROM citus_nodes;

DROP ROLE alper;
```